website/docs/configuration.md

```diff
@@ -526,6 +527,29 @@ Group non-emtpy consecutive layers (zindex) together, so that the shadows and bl
 the whole group instead of each individual layer. This can get rid of some shadowing and blending
 artifacts, but cause worse problems like [#2574](https://github.com/neovide/neovide/issues/2574).
 
+#### Box Drawing
+
+The unicode standard defines several code points that are useful to draw [boxes, diagrams or are
+otherwise decorations][https://en.wikipedia.org/wiki/Box_Drawing]. A font file can include graphical representation for several of these code points
+(glyphs). For example, [Nerd Fonts][https://www.nerdfonts.com/] is a collection of font faces that have been
+patched to include glyphs for several box drawing code points (and many other use-cases).
+
+When Neovide renders these glyphs, some glyphs might not line up correctly or might have
+gaps between adjacent cells, breaking visual continuity. This is especially pronounced when using the [linespace][#line-spacing] configuration option
+to add spacing between lines.
+
+Neovide has support for native rendering (i.e ignore the glyph data in the font) for a subset of these glyphs to avoid this problem. You can enable this via:
+
+```config.toml
+[box-drawing.mode]
```

Comment:
I think this could be just `box-drawing`, or maybe even `boxdrawing`.

And type could be renamed to `mode`. 

It's a bit redundant to have both `type` and `mode`